sessions/js-variables-and-numbers/social-media-post/js/index.js

```diff
@@ -0,0 +1,22 @@
+console.clear();
+
+/*
+1. Create the data for a little social media post. Create variables for the following data using the const keyword:
```

Comment:
I'd probably suggest "small" instead of "little" :)

Perhaps you'd also like to clarify that the intent is to create a variable for each data point.
Like so: "Create a const variable for each data point listed below."

Or something.

Reply:
Good idea